footprint: [[-0.305, -0.18], [-0.305, 0.18], [0.305, 0.18], [0.305, -0.18]]
footprint_padding: 0.05

robot_base_frame: base_link
#update_frequency: 4.0
#publish_frequency: 3.0
transform_tolerance: 0.5

resolution: 0.05

# obstacle detect range
obstacle_range: 4.0
raytrace_range: 10.0

#layer definitions
static:
    map_topic: /plan_map
    subscribe_to_updates: true

obstacles_laser:
    observation_sources: laser
    laser: {data_type: LaserScan, clearing: true, marking: true, topic: /scan, inf_is_valid: true}

inflation:
    inflation_radius: 0.48 # bigger at large field
    cost_scaling_factor: 6.0
